packages/esm-commons-lib/src/utils/encounter-list-config-builder.ts

```diff
@@ -109,6 +109,20 @@ export const getTabColumns = (columnsDefinition: Array<ColumnDefinition>) => {
             }
           });
         }
+        if (column?.conditionalActionOptions?.length) {
```

Comment:
This condition is similar to the one up, why can't you move that logic up or if anything is different you might wanna add it to the condition.